stereovision/ui_utils.py

```diff
@@ -136,8 +136,9 @@ def _set_value(self, parameter, new_value):
         try:
             self.block_matcher.__setattr__(parameter, new_value)
         except BadBlockMatcherArgumentError:
+            print('Bad Block')
             return
-        self.update_disparity_map()
+        #self.update_disparity_map()
```

Comment:
Regarding printing: I'll weed that out. Too few commits while modifying the code, and at the end of the day, I did not review the changes properly. There shall be no printing. It was a remnant from debugging and understanding the information.

Regarding the commented update of the disparity map in the tuning application: In your current master version, the disparity map gets an update from the first change of parameter, and then locks up. I tried to changed the logic as a workaround, so that it only updates on a key press. Entering 'q' will quit the application.

I admit that this pull request was a bit rushed. I will look into the Gnurfos changes and see if they work well for me.